docs/configuration/index-config.md

```diff
@@ -410,6 +412,53 @@ field_mappings:
     type: text
 ```
 
+#### concatenate
+
+Quickwit supports mapping the content of multiple fields to a single one. This can be more efficient at query time than
+searching through dozens of `default_search_fields`. It also allow querying inside a json field without knowing the path
+to the field being searched.
+
+```yaml
+name: my_default_field
+type: concatenate
+concatenated_fields:
+  - text # things inside text, tokenized with the `default` tokenizer
+  - resource.author # all fields in resource.author, assuming resource is a `json` field
+  - _dynamic # content of the dynamic field, assuming indexing mode is `dynamic` (see below)
```

Comment:
I am not fond of this.  "_dynamic" is an internal implementation detail. I don't think this is "public API" at this point. I wonder if this should be a bool in the config.

`include_dynamic_fields: true`

or something like that.